src/formatters/otp_formatter.erl

```diff
@@ -9,6 +9,13 @@
 %% the module `erl_syntax'.
 -module(otp_formatter).
 
+-elvis([{elvis_style, atom_naming_convention, #{regex => "^([a-zA-Z][a-z0-9]*_?)*$"}},
+        {elvis_style,
+         dont_repeat_yourself,
+         #{min_complexity => 20}}]).    %% Allow erl_syntax:syntaxTree/0 type spec
```

Comment:
🤔 why does this option always show up? 🤣

Reply:
Keep in mind that this is code copied from OTP.

Reply:
I mean… I added the exception to avoid refactoring.